circle.yml

```diff
@@ -15,10 +15,11 @@ dependencies:
     - "~/.cabal"
 
   override:
+    - rm -rf /home/ubuntu/.go_workspace/src
     - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
     - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
-    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install
-    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install && go get gopkg.in/alecthomas/gometalinter.v1
```

Comment:
@kevgo 

`go get gopkg.in/alecthomas/gometalinter.v1`

Doesn't appear to use the local version. We can install the locally installed version with
```
cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt/vendor/gopkg.in/alecthomas/gometalinter.v1 && go install
```

This was erring for me as glide appears to not have installed some of the dependencies for `gopkg.in/alecthomas/gometalinter.v1`. I tried pulling from a specific commit on gometalinter and got the compilation to work but then `gometalinter --install` erred for me. It appears the logic for installing the linters is based on it being installed at the top level or being run with a modified GOPATH that is built for `gvt`.

To make this simpler I'd suggest we just remove the locally installed version for now as we aren't using it. We should also update the readme with how to setup the linter locally.